### PR TITLE
Change 'zh' to 'zh-cn'

### LINKED ISSUE #769: Update Chinese lang ID: `zh` → `zh-cn`
Recently, Docsy has renamed language 'zh' to 'zh-cn'.
(Ref: https://github.com/google/docsy/pull/832)

And after [updating Docsy](https://github.com/kubernetes/website/pull/32906) used by [kubernetes/website repo](https://github.com/kubernetes/website),
this causes some issues (like [this](https://github.com/kubernetes/website/issues/33002), and so on..).

To resolve this, [Qiming Teng](https://github.com/tengqm), an active English & Chinese approver in kubernetes/website repo, suggested a solution: 
**update the Chinese language identifier from `zh` to `zh-cn`**,
causing everything under `https//kubernetes.io/zh/*` to be moved to `https//kubernetes.io/zh-cn/*` .
- https://github.com/kubernetes/website/pull/33050
- https://github.com/kubernetes/website/pull/33051

It is likely that we update our Docsy version some time or other, and that we will encounter this issue too.

So I suggest to update the Chinese language identifier from `zh` to `zh-cn` in CNCF Glossary project, before `dev-zh` goes live,
because updating **before** `dev-zh` going live will be better than updating **after** `dev-zh` going live.
(Which means that Chinese contents will be live in `https://glossary.cncf.io/zh-cn/`, not `https://glossary.cncf.io/zh/`.)

How about your thoughts,
Chinese approvers (@hanyuancheung @Jacob953 @Rocksnake @Submarinee) and
maintainers and English approvers (@cjyabraham @CathPag @jasonmorgan @seokho-son @iamNoah1) ? 😊

[Refs]
- https://github.com/google/docsy/issues/732
- https://github.com/google/docsy/pull/712
- https://github.com/google/docsy/pull/826
- https://github.com/google/docsy/pull/832
- https://github.com/kubernetes/website/issues/33002

### PR DESCRIPTION
**Pull request by jihoon-seo**
- Closes #769
- Update some links
- Update `_index.md` with adding localization contributors (commits by @Jacob953)

### CODE REVIEW

**Review by hanyuancheung — APPROVED** [review]
lgtm. 

@Submarinee @Jacob953 @Rocksnake pls pay attention, Chinese team, don't forget to check if there still are these expired words or paths. If so, pls change them into the new ones.

**Review by Jacob953 — APPROVED** [review]
LGTM. Thx for your help! @jihoon-seo

File: content/zh-cn/_index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+云原生词汇表是由 CNCF 商业价值小组委员会 (BVS, Business Value Subcommittee) 领导的一个项目。该项目目的是在不需要先验技术知识的情况下，以清晰通俗的语言解释云原生概念。
+
+<p><img class="mt-5" src="/images/homepage/stage.jpg" alt="A woman and two men presenting technical info on a stage"></p>
+
+## 社区贡献
+
+对本机词汇表的更改、添加和改进对所有人开放。我们使用由 CNCF 管理的社区驱动流程来开发和完善这个共享词典。该词汇表提供了一个厂商中立的平台，用于管理云原生技术的共享词汇表。欢迎所有参与者根据本项目的目的和章程作出贡献。
+
+任何想要贡献的人都可以提交 GitHub issue 或 创建 PR。欲了解更多信息，请查看 [如何贡献](/zh-cn/contribute/) 和遵循 [样式指南](/zh-cn/style-guide/)，并加入 CNCF Slack 中的 #glossary 频道。这里有 #glossary-localizations 频道，供想要将 glossarg 翻译成母语的人使用。
```
Comment (jihoon-seo) [inline content/zh-cn/_index.md:16]: Please fix this typo. 😊

```suggestion
任何想要贡献的人都可以提交 GitHub issue 或 创建 PR。欲了解更多信息，请查看 [如何贡献](/zh-cn/contribute/) 和遵循 [样式指南](/zh-cn/style-guide/)，并加入 CNCF Slack 中的 #glossary 频道。这里有 #glossary-localizations 频道，供想要将 glossary 翻译成母语的人使用。
```

File: content/zh-cn/_index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## 致谢
+
+云原生词汇表可从 CNCF 营销委员会 (CNCF Marketing Committee) (商业价值小组委员会) 获得,
+包括 [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) 以及更多的贡献者. 如想获得完整的贡献者名单, 请跳转至 [GitHub Pages 页](https://github.com/cncf/glossary/graphs/contributors).
+
+中文版的云原生词汇表由 [Chester Cheung](https://github.com/hanyuancheung/), [Brian Yan
+](https://github.com/Rocksnake/), [Jacob953 Yu](https://github.com/Jacob953/), [
+Submarinee](https://github.com/Submarinee/) 和其他贡献者维护。
```
Comment (jihoon-seo) [inline content/zh-cn/_index.md:26]: In preview, there exists an unwanted space:
image

---

So, some suggestion for 'initial localization contributors list' section. 😊

```suggestion
中文版的云原生词汇表由 [Chester Cheung](https://github.com/hanyuancheung), 
[Brian Yan](https://github.com/Rocksnake), 
[Jacob953 Yu](https://github.com/Jacob953), 
[Submarinee](https://github.com/Submarinee) 和其他贡献者维护。
```

**Review by jihoon-seo — COMMENTED** [review]
@Jacob953 PTAL!

**Review by Rocksnake — APPROVED** [review]
lgtm

**Comment by Jacob953** [comment]
Hi, @jihoon-seo PTAL~
I've updated links, fixed typos, and localized style-guide/_index.md at https://github.com/jihoon-seo/cncf-glossary/pull/8

Please let me know if it needs to be updated. ;)

**Review by seokho-son — APPROVED** [review]
LGTM

File: content/zh-cn/distributed_apps.md [hunk 3]
```diff
@@ -19,5 +19,5 @@ category: 概念
 ## 如何帮助
 
 当把一个应用程序拆分成不同的部分并在许多地方运行时，整个系统可以容忍更多的故障。
-它还允许应用程序利用单个应用程序实例所不具备的扩展功能，即 [水平扩展](/horizontal_scaling/)的能力。
+它还允许应用程序利用单个应用程序实例所不具备的扩展功能，即 [水平扩展](/zh-cn/horizontal_scaling/)的能力。
```
Comment (jihoon-seo) [inline content/zh-cn/distributed_apps.md:22]: The term "Horizontal Scaling" is not translated into Chinese yet.
(Currently translated terms: https://github.com/cncf/glossary/tree/dev-zh/content/zh-cn)
So I suggest to keep `[水平扩展](/horizontal_scaling/)`.

File: content/zh-cn/cloud_native_apps.md [hunk 4]
```diff
@@ -6,15 +6,15 @@ category: 概念
...
+云原生应用程序专门设计用于利用 [云计算](/zh-cn/cloud_computing/) 中的创新。
+这些应用程序可以轻松地与其各自的云架构集成，充分利用云的资源和 [可扩展性](/zh-cn/scalability/) 功能。
 它还指利用云计算驱动的基础设施创新的应用程序。
 今天的云原生应用程序包括在云提供商的数据中心和本地云原生平台上运行的应用程序。
 
 ## Problem it addresses
 
 传统上，本地环境以相当定制的方式提供计算资源。
-每个数据中心都有与特定环境 [紧密耦合](/tightly_coupled_architectures/) 应用程序的服务，通常严重依赖于基础设施的手动配置，例如 [虚拟机](/zh/virtual_machine/) 和服务。
+每个数据中心都有与特定环境 [紧密耦合](/zh-cn/tightly_coupled_architectures/) 应用程序的服务，通常严重依赖于基础设施的手动配置，例如 [虚拟机](/zh-cn/virtual_machine/) 和服务。
```
Comment (jihoon-seo) [inline content/zh-cn/cloud_native_apps.md:17]: The term "Tightly Coupled Architectures" is not translated into Chinese yet.
So I suggest to keep `[紧密耦合](/tightly_coupled_architectures/)`.

File: content/zh-cn/cloud_native_apps.md [hunk 5]
```diff
@@ -23,5 +23,5 @@ category: 概念
 
 虽然云原生应用程序没有“一刀切”的路径，但它们确实有一些共性。
 云原生应用程序具有弹性、可管理性，并由配套的云服务套件提供帮助。
-各种云服务实现了高度的 [可观察性](/observability/)，使用户能够在问题升级之前检测和解决问题。
+各种云服务实现了高度的 [可观察性](/zh-cn/observability/)，使用户能够在问题升级之前检测和解决问题。
```
Comment (jihoon-seo) [inline content/zh-cn/cloud_native_apps.md:26]: The term "Observability" is not translated into Chinese yet.
So I suggest to keep `[可观察性](/observability/)`.

File: content/zh-cn/cloud_native_security.md [hunk 6]
```diff
@@ -6,10 +6,10 @@ category: 概念
 
 ## 是什么
 
-云原生安全是一种将安全性构建到 [云原生应用程序](/zh/cloud_native_apps/) 中的方法。
+云原生安全是一种将安全性构建到 [云原生应用程序](/zh-cn/cloud_native_apps/) 中的方法。
 它确保安全是从开发到生产的整个应用程序生命周期的一部分。
 云原生安全旨在确保与传统安全模型相同的标准，同时适应云原生环境的特殊性，即快速的代码更改和高度短暂的基础设施。
-云原生安全与称为 [测试左移](/devsecops/) 的实践高度相关。
+云原生安全与称为 [测试左移](/zh-cn/devsecops/) 的实践高度相关。
```
Comment (jihoon-seo) [inline content/zh-cn/cloud_native_security.md:12]: The term "DevSecOps" is not translated into Chinese yet.
So I suggest to keep `[测试左移](/devsecops/)`.

File: content/zh-cn/containerization.md [hunk 7]
```diff
@@ -6,15 +6,15 @@ category: 技术
...
 容器构建过程需要遵守 [开放容器倡议](https://opencontainers.org)(OCI) 标准。
 只要输出的是一个符合这个标准的容器镜像，使用哪种容器化工具并不重要。
 
 ## 解决的问题 
 
-在容器盛行之前，企业依靠虚拟机 (VMs) 在一台 [裸机](/bare_metal_machine/) 上协调多个应用程序。
+在容器盛行之前，企业依靠虚拟机 (VMs) 在一台 [裸机](/zh-cn/bare_metal_machine/) 上协调多个应用程序。
 虚拟机比容器大得多，需要一个管理程序来运行。由于这些较大的虚拟机模板的存储、备份和传输，创建虚拟机模板也很慢。
-此外，虚拟机可能会出现配置漂移，这违反了 [不变性](/immutable_infrastructure/) 的原则。
+此外，虚拟机可能会出现配置漂移，这违反了 [不变性](/zh-cn/immutable_infrastructure/) 的原则。
```
Comment (jihoon-seo) [inline content/zh-cn/containerization.md:17]: `(/bare_metal_machine/)` and `(/immutable_infrastructure/)` too. 😊

File: content/zh-cn/containers_as_a_service.md [hunk 8]
```diff
@@ -6,13 +6,13 @@ category: 技术
 
 ## 是什么
 
-容器即服务（CaaS）是一种云服务，有助于使用基于 [容器](/zh/container/) 的 [抽象](/abstraction/) 管理和部署应用程序。
+容器即服务（CaaS）是一种云服务，有助于使用基于 [容器](/zh-cn/container/) 的 [抽象](/zh-cn/abstraction/) 管理和部署应用程序。
```
Comment (jihoon-seo) [inline content/zh-cn/containers_as_a_service.md:9]: `(/abstraction/)` too. 😊

File: content/zh-cn/containers_as_a_service.md [hunk 9]
```diff
@@ -6,13 +6,13 @@ category: 技术
...
 这项服务可以部署在企业内部或云中。
 
 CaaS 供应商提供了一个框架或协调平台，使容器部署和管理的关键 IT 功能自动化。
-它帮助开发者建立安全和 [可扩展](/scalibility/) 的容器化应用。
+它帮助开发者建立安全和 [可扩展](/zh-cn/scalibility/) 的容器化应用。
```
Comment (jihoon-seo) [inline content/zh-cn/containers_as_a_service.md:13]: Please fix the typo which makes the link be broken. 😊

`scalibility` → `scalability`

File: content/zh-cn/containers_as_a_service.md [hunk 10]
```diff
@@ -21,6 +21,6 @@ CaaS 供应商提供了一个框架或协调平台，使容器部署和管理的
 ## 如何帮助
 
 当把容器化应用部署到 CaaS 平台时，用户可以通过日志聚合和监控工具获得对系统性能的可见性。
-CaaS 还包括 [自动扩展](/auto_scaling/) 和协调管理的内置功能。
-它使团队能够建立高可见性和高可用性的 [分布式系统](/zh/distributed_systems/)。
-此外，通过允许快速部署，CaaS 提高了团队的开发速度。在容器确保一致的部署目标的同时，CaaS 通过减少管理部署所需的 [DevOps](/zh/devops/) 资源，降低了工程运营成本。
+CaaS 还包括 [自动扩展](/zh-cn/auto_scaling/) 和协调管理的内置功能。
```
Comment (jihoon-seo) [inline content/zh-cn/containers_as_a_service.md:24]: `(/auto_scaling/)` too. 😊

File: content/zh-cn/devops.md [hunk 11]
```diff
@@ -8,7 +8,7 @@ category: 概念
 开发运维是一种方法论，其中团队拥有从应用程序开发到生产操作的整个过程，因此 DevOps，它不仅仅是实施一套技术，还需要文化和流程的彻底转变。 DevOps 需要一组工程师来处理小组件（相对于整个功能），从而减少交接——一个常见的错误来源。
 
 ## 强调的问题
-传统上，在具有 [紧密耦合](/tightly_coupled_architectures/) [单体应用程序](/zh/monolithic_apps/) 的复杂组织中，工作通常分散在多个组之间。 这导致了多次交接和较长的交货时间。 每次组件或更新准备就绪时，都会将其放入队列中以供下一个团队使用。 因为个人只参与了项目的一小部分，这种方法导致缺乏所有权。 他们的目标是将工作交给下一个小组，而不是为客户提供正确的功能——明显的优先级错位。
+传统上，在具有 [紧密耦合](/zh-cn/tightly_coupled_architectures/) [单体应用程序](/zh-cn/monolithic_apps/) 的复杂组织中，工作通常分散在多个组之间。 这导致了多次交接和较长的交货时间。 每次组件或更新准备就绪时，都会将其放入队列中以供下一个团队使用。 因为个人只参与了项目的一小部分，这种方法导致缺乏所有权。 他们的目标是将工作交给下一个小组，而不是为客户提供正确的功能——明显的优先级错位。
```
Comment (jihoon-seo) [inline content/zh-cn/devops.md:11]: `(/tightly_coupled_architectures/)` too. 😊

File: content/zh-cn/distributed_systems.md [hunk 12]
```diff
@@ -7,7 +7,7 @@ category: 概念
 ## 是什么
 
 分布式系统是通过网络连接的自主计算元素的集合，在用户看来是一个单一的连贯系统。
-一般被称为 [节点](/nodes/)，这些组件可以是硬件设备（如电脑、手机）或软件进程。
+一般被称为 [节点](/zh-cn/nodes/)，这些组件可以是硬件设备（如电脑、手机）或软件进程。
```
Comment (jihoon-seo) [inline content/zh-cn/distributed_systems.md:10]: `(/nodes/)` too. 😊

File: content/zh-cn/distributed_systems.md [hunk 13]
```diff
@@ -16,11 +16,11 @@ category: 概念
...
 如果没有分布式计算，我们今天依赖的许多应用就不可能实现。
 
-传统上，系统会纵向 [扩展](/zh/scalability/)。这就是当你在一台单独的机器上添加更多的 CPU 或内存。
+传统上，系统会纵向 [扩展](/zh-cn/scalability/)。这就是当你在一台单独的机器上添加更多的 CPU 或内存。
 垂直扩展很耗时，需要停机，而且很快就会达到极限。
 
 ## 如何帮助
 
-分布式系统允许 [水平扩展](/horizontal_scaling/)（例如，在需要时向系统添加更多节点）。这可以是自动化的，允许系统处理工作负荷或资源消耗的突然增加。
+分布式系统允许 [水平扩展](/zh-cn/horizontal_scaling/)（例如，在需要时向系统添加更多节点）。这可以是自动化的，允许系统处理工作负荷或资源消耗的突然增加。
```
Comment (jihoon-seo) [inline content/zh-cn/distributed_systems.md:24]: `(/horizontal_scaling/)` too. 😊

File: content/zh-cn/kubernetes.md [hunk 14]
```diff
@@ -7,20 +7,20 @@ category: 技术
 ## 是什么
 
 Kubernetes，通常缩写为K8s，是一种流行的现代基础设施自动化的开源工具。
-它就像一个数据中心的操作系统，管理在 [分布式系统](/zh/distributed_systems/) 上运行的应用程序（就像你笔记本上的操作系统，管理你的应用程序）。
+它就像一个数据中心的操作系统，管理在 [分布式系统](/zh-cn/distributed_systems/) 上运行的应用程序（就像你笔记本上的操作系统，管理你的应用程序）。
 
-Kubernetes在 [集群](/zh/cluster/) 的 [节点](/nodes/) 上调度 [容器](/zh/container/)。
+Kubernetes在 [集群](/zh-cn/cluster/) 的 [节点](/zh-cn/nodes/) 上调度 [容器](/zh-cn/container/)。
```
Comment (jihoon-seo) [inline content/zh-cn/kubernetes.md:12]: `(/nodes/)` too. 😊

File: content/zh-cn/kubernetes.md [hunk 15]
```diff
@@ -7,20 +7,20 @@ category: 技术
...
 它捆绑了几个基础设施结构，有时被称为 "基元"，如应用程序的实例、负载平衡器、持久性存储等，以一种可以被组成应用程序的方式。
 
 Kubernetes 实现了自动化和可扩展性，使用户能够以可重复的方式声明性地部署应用程序。
-Kubernetes 生态系统中的软件产品和项目利用这种自动化和可扩展性来扩展 Kubernetes [API](/application_programming_interface/) 。
+Kubernetes 生态系统中的软件产品和项目利用这种自动化和可扩展性来扩展 Kubernetes [API](/zh-cn/application_programming_interface/) 。
```
Comment (jihoon-seo) [inline content/zh-cn/kubernetes.md:16]: `(/application_programming_interface/)` too. 😊

File: content/zh-cn/microservices.md [hunk 16]
```diff
@@ -8,17 +8,17 @@ category: 概念
...
+简而言之，微服务指的是一种应用架构模式，通常与 [单体应用](/zh-cn/monolithic_apps/) 形成对比。
 
 ## 解决的问题
 
-微服务是对单体应用所带来的挑战的一种回应。一般来说，一个应用程序的不同部分需要分别进行 [扩展](/zh/scalability/)。
+微服务是对单体应用所带来的挑战的一种回应。一般来说，一个应用程序的不同部分需要分别进行 [扩展](/zh-cn/scalability/)。
 例如，一个在线商店将有更多的产品视图而不是结账。这意味着你需要更多的产品视图功能的运行，而不是结账。
 在一个单一的应用程序中，这些逻辑位不能被单独部署。如果你不能单独扩展产品功能，你将不得不复制整个应用程序和所有其他你不需要的组件--这是一种低效的资源利用。
-单机式应用程序也使开发人员容易屈服于设计陷阱。因为所有的代码都在一个地方，所以更容易使这些代码 [高耦合](/tightly_coupled_architectures/)，更难执行关注点分离的原则。
+单机式应用程序也使开发人员容易屈服于设计陷阱。因为所有的代码都在一个地方，所以更容易使这些代码 [高耦合](/zh-cn/tightly_coupled_architectures/)，更难执行关注点分离的原则。
```
Comment (jihoon-seo) [inline content/zh-cn/microservices.md:18]: `(/tightly_coupled_architectures/)` too. 😊

File: content/zh-cn/scalability.md [hunk 17]
```diff
@@ -4,6 +4,6 @@ status: Completed
 category: 属性
 ---
 
-可扩展性指的是一个系统能有多大的发展。这就是增加做任何系统应该做的事情的能力。 例如，[Kubernetes](/zh/kubernetes/) [集群](/zh/cluster/) 通过增加或减少 [容器化](/zh/containerization/) 应用程序的数量来进行扩展，但这种可扩展性取决于几个因素。 它有多少[节点](/nodes/)，每个节点可以处理多少个[容器](/container/)，控制平面可以支持多少条记录和操作？
+可扩展性指的是一个系统能有多大的发展。这就是增加做任何系统应该做的事情的能力。 例如，[Kubernetes](/zh-cn/kubernetes/) [集群](/zh/cluster/) 通过增加或减少 [容器化](/zh-cn/containerization/) 应用程序的数量来进行扩展，但这种可扩展性取决于几个因素。 它有多少[节点](/zh-cn/nodes/)，每个节点可以处理多少个[容器](/zh-cn/container/)，控制平面可以支持多少条记录和操作？
```
Comment (jihoon-seo) [inline content/zh-cn/scalability.md:7]: `(/nodes/)` too. 😊

File: content/zh-cn/scalability.md [hunk 18]
```diff
@@ -4,6 +4,6 @@ status: Completed
...
 
-可扩展的系统使添加更多容量更容易。 主要有两种缩放方法。 一方面，有 [水平缩放](/horizontal_scaling/) 添加更多节点来处理增加的负载。 相比之下，在 [垂直缩放](/vertical_scaling/) 中，单个节点的功能更强大，可以执行更多事务（例如，通过向单个机器添加更多内存或 CPU）。 可扩展的系统能够轻松更改并满足用户需求。
+可扩展的系统使添加更多容量更容易。 主要有两种缩放方法。 一方面，有 [水平缩放](/zh-cn/horizontal_scaling/) 添加更多节点来处理增加的负载。 相比之下，在 [垂直缩放](/zh-cn/vertical_scaling/) 中，单个节点的功能更强大，可以执行更多事务（例如，通过向单个机器添加更多内存或 CPU）。 可扩展的系统能够轻松更改并满足用户需求。
```
Comment (jihoon-seo) [inline content/zh-cn/scalability.md:9]: `(/horizontal_scaling/)` and `(/vertical_scaling/)` too. 😊

File: content/zh-cn/serverless.md [hunk 19]
```diff
@@ -5,10 +5,10 @@ Category: Technology
 ---
 
 ## 是什么
-Serverless 是一种云原生开发模型，允许开发人员构建和运行应用程序，而无需管理服务器。 Serverless 中仍有服务器，但它们被 [抽象](/abstraction/) 出来，远离应用程序开发。 云提供商处理配置、维护和 [扩展](/zh/scalability/) 服务器基础架构的日常工作。 开发人员可以简单地将他们的代码打包在 [容器](/zh/container/) 中进行部署。 部署后，Serverless 应用程序会响应需求并根据需要自动扩展和缩减。 公共云提供商的 Serverless 产品通常通过事件驱动的执行模型按需计量。 因此，当无服务器功能处于空闲状态时，它不会花费任何费用。
+Serverless 是一种云原生开发模型，允许开发人员构建和运行应用程序，而无需管理服务器。 Serverless 中仍有服务器，但它们被 [抽象](/zh-cn/abstraction/) 出来，远离应用程序开发。 云提供商处理配置、维护和 [扩展](/zh-cn/scalability/) 服务器基础架构的日常工作。 开发人员可以简单地将他们的代码打包在 [容器](/zh-cn/container/) 中进行部署。 部署后，Serverless 应用程序会响应需求并根据需要自动扩展和缩减。 公共云提供商的 Serverless 产品通常通过事件驱动的执行模型按需计量。 因此，当无服务器功能处于空闲状态时，它不会花费任何费用。
```
Comment (jihoon-seo) [inline content/zh-cn/serverless.md:8]: `(/abstraction/)` too. 😊

File: content/zh-cn/serverless.md [hunk 20]
```diff
@@ -5,10 +5,10 @@ Category: Technology
...
 
 ## 解决的问题
-在标准的 [基础设施即服务 (IaaS)](/infrastructure_as_a_service/) [云计算](/zh/cloud_computing/) 模型下，用户预先购买容量单位，这意味着您需要向公共云提供商支付永远在线的服务器组件的费用来运行您的应用程序。
+在标准的 [基础设施即服务 (IaaS)](/zh-cn/infrastructure_as_a_service/) [云计算](/zh-cn/cloud_computing/) 模型下，用户预先购买容量单位，这意味着您需要向公共云提供商支付永远在线的服务器组件的费用来运行您的应用程序。
```
Comment (jihoon-seo) [inline content/zh-cn/serverless.md:11]: `(/infrastructure_as_a_service/)` too. 😊

File: content/zh-cn/service_mesh.md [hunk 21]
```diff
@@ -6,9 +6,9 @@ category: 技术
 
 ## 是什么
 
-在 [微服务](/zh/microservices/) 的理念里，应用程序被分解成多个较小的 [服务](/service/)，通过网络进行通信。
+在 [微服务](/zh-cn/microservices/) 的理念里，应用程序被分解成多个较小的 [服务](/zh-cn/service/)，通过网络进行通信。
```
Comment (jihoon-seo) [inline content/zh-cn/service_mesh.md:9]: `(/service/)` too. 😊

File: content/zh-cn/service_mesh.md [hunk 22]
```diff
@@ -6,9 +6,9 @@ category: 技术
...
 就像你的 WIFI 网络一样，计算机网络本质上是不可靠的，可被黑客攻击的，而且往往很慢。
-服务网格通过管理服务之间的流量（即通信），并在所有服务中统一添加 [可靠性](/zh/reliability/)、[可观察性](/observability/) 和安全功能来解决这一系列新的挑战。
+服务网格通过管理服务之间的流量（即通信），并在所有服务中统一添加 [可靠性](/zh-cn/reliability/)、[可观察性](/zh-cn/observability/) 和安全功能来解决这一系列新的挑战。
```
Comment (jihoon-seo) [inline content/zh-cn/service_mesh.md:11]: `(/observability/)` too. 😊

File: content/zh-cn/service_proxy.md [hunk 23]
```diff
@@ -6,7 +6,7 @@ category: 技术
 
 ## 是什么
 
-服务代理拦截进出某项 [服务](/service/) 的流量，对其应用一些逻辑，然后将该流量转发给另一项服务。
+服务代理拦截进出某项 [服务](/zh-cn/service/) 的流量，对其应用一些逻辑，然后将该流量转发给另一项服务。
```
Comment (jihoon-seo) [inline content/zh-cn/service_proxy.md:9]: `(/service/)` too. 😊

**Review by jihoon-seo — COMMENTED** [review]
@Jacob953 PTAL! 😊

**Pull request merged**